src/main/java/org/swisspush/redisques/util/RedisquesConfiguration.java

```diff
@@ -704,6 +704,8 @@ public String toString() {
         return asJsonObject().toString();
     }
 
+    public boolean isDequeueStatsEnabled() { return getDequeueStatisticReportIntervalSec()>0; }
```

Comment:
look for code format like spaces, indention and bracket placement

Reply:
Fixed!